tldw_Server_API/tests/Chat_NEW/integration/test_moderation.py

```
@@ -192,6 +213,74 @@ def test_input_block_returns_400(monkeypatch):
+                    "model": "gpt-4o-mini",
+                    "messages": [{"role": "user", "content": "please say secret"}],
+                    "stream": False,
+                }
+                r = _post_with_csrf(client, "/api/v1/chat/completions", json=body, headers=_auth_headers(client))
+                assert r.status_code == 503
+                assert "mandatory audit persistence unavailable" in r.json().get("detail", "").lower()
+                assert not mock_provider.called
+    finally:
+        try:
+            os.unlink(db_path)
+            if os.path.exists(db_path + "-wal"): os.unlink(db_path + "-wal")
```

Ruff (0.15.9)

[error] Multiple statements on one line (colon)

(E701)